ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1093 @@
+----
+ARJUNA016036: commit on < formatId=131077, gtrid_length=35, bqual_length=43, tx_uid=..., node_name=server1, branch_uid=..., subordinatenodename=server2, eis_name=unknown eis name > (org.jboss.as.quickstarts.ejb.mock.MockXAResource@731ae22) failed with exception $XAException.XAER_RMFAIL: javax.transaction.xa.XAException
+----
+
+The quickstart intentionally setup the exception to simulate
+the failure during the commit. The log reports that
+the `MockXAResource` threw the `XAException.XAER_RMFAIL` error, which is the
+intermittent failure that can be recovered later.
+
+Use telnet to force recovery processing.
+
+[source]
+----
+telnet localhost 4712
+Trying 127.0.0.1...
+Connected to localhost.
+Escape character is '^]'.
+SCAN
+DONE
+Connection closed by foreign host.
+----
+
+The recovery manager  on `server1` takes some time to process the request.
+When processing is complete, `DONE` is printed and telnet exits. At that point, all work is committed.
+
+The number of commits on the server increases by one.
+
+The whole sequence of the calls in your terminal resemble the following example code.
+
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+# HTTP endpoind invocation
+curl http://localhost:8080/client/remote-outbound-fail-stateless
+# output:
+# host: mydev.narayana.io/192.168.0.1, jboss node name: server2
+
+# the EJB remote call went to server2 which is started at port 8180
+# examining the number of commits at server2 then
+curl http://localhost:8180/server/commits; echo
+# output:
+# ["host: mydev.narayana.io/192.168.0.1, jboss node name: server2","1"]
+
+# the commit count remains the same as it was before the call
+# running recovery
+telnet localhost 4712
+# type 'SCAN' and wait for finishing
+
+# the number of commits increased by one
+curl http://localhost:8180/server/commits; echo
+# output:
+# ["host: mydev.narayana.io/192.168.0.1, jboss node name: server2","2"]
+----
+
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
+
+Repeat the same for the `server2` and `server3` by navigating
+to the quickstart sub-folder `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server` and run:
+
+[source,sh,options="nowrap"]
+----
+mvn wildfly:undeploy -Dwildfly.port=10090
+mvn wildfly:undeploy -Dwildfly.port=10190
+----
+
+== Server Log: Expected Warnings and Errors
+
+This quickstart is not production grade. The server log includes the following warnings 
+during startup. It is safe to ignore these warnings.
+
+[source,options="nowrap"]
+----
+HHH000431: Unable to determine H2 database version, certain features may not work
+
+WFLYDM0111: Keystore standalone/configuration/application.keystore not found, it will be auto generated on first use with a self signed certificate for host localhost
+
+WFLYSRV0018: Deployment "deployment.server.war" is using a private module ("org.jboss.jts") which may be changed or removed in future versions without notice.
+----
+
+== Kubernetes/OpenShift deployment
```

Comment:
We support Quickstarts on Openshift only (s2i or bootable jar) so I suggest you stick to that and drop running on Kubernetes